packages/paste-design-tokens/tokens/global/border-color.yml

```diff
@@ -5,15 +5,18 @@ imports:
   - ../aliases/color.yml
   - ../aliases/color-palette.yml
 props:
+  color-border-dark:
```

Comment:
What is dark for? Do we use it? Was it in the ticket?

Reply:
Looks like we're using it in Checkbox. I just moved the order of the tokens around to go from dark -> light.

Reply:
OMG, yup. I see it now 🤦